tasks/main.yml

```diff
@@ -1,48 +1,43 @@
 ---
 # This Playbook configure firewall 'ferm'
 - name: Ferm | Install iptables
-  action:
-    module: "{{ ansible_pkg_mgr }}"
+  package:
     name: iptables
     state: present
 
 - name: Ferm | Install Ferm
-  action:
-    module: "{{ ansible_pkg_mgr }}"
+  package:
     name: ferm
     state: present
 
 - name: Ferm | Disable firewalls
-  service: name={{ item }} state=stopped enabled=no
-  with_items: disable_other_firewalls
+  service: name="{{ item }}" state=stopped enabled=no
+  with_items: "{{ disable_other_firewalls }}"
   ignore_errors: true
 
 - name: Ferm | Display rules var
   debug: var=ferm_rules
 
-- name: Ferm | Make sure the directory for firewall rules exist
-  file: path=/etc/ferm state=directory owner=root group=root
-
 - name: Ferm | Create config directory
-  file: path=/etc/ferm/ferm.d state=directory owner=root group=root
+  file: path=/etc/ferm.d state=directory owner=root group=root
```

Comment:
I would like to keep ferm.d under etc/ferm if the directory exists:
for non-redhat it will create:
```
/etc/ferm.d
/etc/ferm/ferm.conf
```

I propose to create facts, eg:
```
set_fatct:
  fermd_dir: >-
     {% if ansible_os_family == "RedHat" %} 
     /etc/ferm.d
     {% else %} 
     /etc/ferm/ferm.d
```